scrapers/HotWifeXXX.yml

```diff
@@ -0,0 +1,26 @@
+name: "HotWifeXXX"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.hotwifexxx.com/tour_hwxxx/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="trailerMInfo"]
+    scene:
+      Title: $info//h2/text()
+      Date:
+        selector: $info//div[@class="released2 trailerStarr"]/text()
+        replace:
+          - regex: (,.+)
+            with:
+        parseDate: 01/02/2006
+      Details: $info//div[@class="videoDetails"]/p/text()
+      Tags:
```

Comment:
since `Tags:` is not used it should be removed

Reply:
Fixed!